clients/admin-ui/src/features/dataset/DatasetTable.tsx

```diff
@@ -31,6 +34,8 @@ const DatasetsTable = () => {
       dispatch(setActiveDatasetFidesKey(undefined));
```

Comment:
I completely agree there, great point! 🙌🏽

Reply:
While removing this it brought up another potential issue that I wasn't sure about... When generating a dataset (and not classifying) - when the `generate` call completes the first dataset is loaded instead of landing at the default dataset view. If classifying, we hold on the dataset view until it is done processing (love the change to have it pull for updates 🙌🏽 )

I'm assuming it must have forced a click function on the first row somewhere to enable the load dataset button but I could be mistaken 🤔

Reply:
I think it would call generate + create, then [navigate to the individual dataset page](https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/dataset/DatabaseConnectForm.tsx#L188) whereas a classify will [navigate you back to the index page](https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/dataset/DatabaseConnectForm.tsx/#L204)

It looks like that's the intended behavior, does that line up with what you were concerned about?